transcom/mymove
# B 18050 ppm ssw form pdf replacement - PDFCPU Update 0.6.0

## B 18050 ppm ssw form pdf replacement 

## Summary

This ticket requires the PDFCPU Library to be upgraded from 0.2.5 -> 0.6.0. This was a breaking update, but all functionality should be present and all tests pass after some manual adjustments were made.

Is there anything you would like reviewers to give additional scrutiny?  - All PDF functions

[this article](tbd) explains more about the approach used. - Library was updated, functions and tests were manually reconfigured to continue working as they were before the update.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the client
2. Login as a user
3. Upload orders
4. Login as office user
5. Verify PDF Orders are present

### Frontend


### Backend
